app.js

```diff
@@ -0,0 +1,40 @@
+// 1. Importaciones
+
+const express = require ("express")
+
+const app = express ()
+
+require ("dotenv").config()
+
+// 2. Middlewares
+
+app.use(express.static('public'))
+
+app.set("views", __dirname + "/views")
+
+app.set("view engine", "hbs")
+
+// 3. Rutas
+
+app.get("/", (req, res) => {
+    res.render("home")
+})
+
+app.get("/about", (req, res) => {
+    res.render("about")
+})
+
+app.get("/works", (req, res) => {
+    res.render("works")
+})
+
+app.get("/photoGallery", (req, res) => {
+    res.render("photoGallery")
+})
+
+
+// 4. Servidor
+
+app.listen (process.env.PORT, () => {
+    console.log(`Servidor activo en puerto ${process.env.PORT}`)
```

Comment:
Con ayuda de la siguiente linea y ctr o cmd + click puedes abrir la ruta desde tu terminal, esto va a ahorrar tiempo en el desarrollo de tus proyectos 😉
```suggestion
console.log(`Servidor activado en el puerto http://localhost:${process.env.PORT}`)
```